src/styles/main.scss

```diff
@@ -0,0 +1,78 @@
+$main-color: #eee;
+$hover-color: #ffbfcb;
+$font-family: Arial, Halvetica, san-serif;
```

Comment:
There's a typo in the font-family declaration; 'Halvetica' should be 'Helvetica'.